docs/src/pages/components/selects/selects-pt.md

```diff
@@ -92,6 +92,12 @@ Como na seleção única, você pode extrair o novo valor acessando `event.targe
 
 {{"demo": "pages/components/selects/ControlledOpenSelect.js"}}
 
+## Abrir automaticamente
+
+É possível fazer com que a seleção abra automaticamente ao ser montada por meio da propriedade `defaultOpen`.
+
+{{"demo": "pages/components/selects.DefaultOpen.js"}}
+
```

Comment:
Translations are done via Crowdin. We don't commit them manually. Please leave just the English version and restore the `-pt` and `-zh` files.

Reply:
updated :)